src/markdown-pages/docs/galasa-hub.md

```diff
@@ -5,6 +5,15 @@ title: "Galasa Hub"
 
 Stay up-to-date with all things Galasa with blogs, podcasts and videos from fellow Galasians. 
 
+## Blog: Goodbye Manual z/OS Scenarios…Hello Galasa! 
+Find out how IBM Z and z/OS Platform Evaluation and Test (zPET) are using Galasa to help them automate their manual scenarios and discover their future plans for Galasa.<br>
```

Comment:
do we need to say the zPET `team`?

Reply:
Yes, I think looking at it now that it would be better to say "the ... team" rather than just the org.  I will add it in.